lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1594,9 +1596,48 @@ void SwiftASTContext::AddExtraClangArgs(const std::vector<std::string> &source,
   }
 }
 
+namespace {
+
+bool HasNonexistentExplicitModule(const std::vector<std::string> &args) {
+  for (const auto &arg : args) {
+    StringRef val = arg;
+    if (!val.consume_front("-fmodule-file="))
+      continue;
+    // The value's format is 'ModuleName=ModulePath'. Drop the prefix up to the
+    // first '=' character to obtain the path.
```

Comment:
Not necessarily. There's another format that's just `-fmodule-file=ModulePath` we might switch to in the future.

Reply:
@jansvoboda11 thanks. Do you know, are explicit modules guaranteed to be absolute paths? I would presume yes.

Reply:
That ultimately depends on the build system. I'd expect them to be absolute in practice, but it's not a guarantee. Theoretically these might be relative to `-working-directory` or the process CWD.